src/InitSettings.jsx

```diff
@@ -24,55 +23,37 @@ const Item = styled(Paper)(({ theme }) => ({
 
 function InitSettings() {
   const {
-    walking, setKinematicValue, setWalkingValue,
+    published, setPublished, setKinematic, setWalking,
   } = useContext(WalkContext);
 
-  const client = useClient();
-  const logger = useLogger();
+  const client = new aruku_interfaces.ConfigClient(`${import.meta.env.GRPC_WEB_API_URL}`, null, null);
```

Comment:
better you define the const variable of GRPC_WEB_API_URL first and then use it, instead of like this